web/init/src/components/config_render/FileInput.jsx

```diff
@@ -46,32 +55,43 @@ export default class FileInput extends React.Component {
   }
 
   render() {
-
     let label;
-    if (this.props.label) {
-      label = this.props.label;
-    } else if (this.props.multiple) {
-      label = "Choose files";
-    } else {
-      label = "Choose file";
-    }
+    this.props.label ? label = this.props.label : this.props.multiple
```

Comment:
Any particular reason for moving from what we had to this notation? In my opinion what we had before was easier to read.

Reply:
Changed it because it looks cleaner for me, but I can add it back how was before :)

Reply:
I mean it's a matter of preference really. I'm fine with the change, just was curious as to why you updated it